Assets/Character/AaronTestAssets/Player.cs

```diff
@@ -0,0 +1,60 @@
+﻿using UnityEngine;
+using System.Collections;
+
+
+
+public class Player : MonoBehaviour {
+
+
+    public float speed = 2f;
+    private Animator animator;
+    private Rigidbody2D rigidbody;
+	// Use this for initialization
+	void Start () {
+
+        animator = GetComponent<Animator>();
+        rigidbody = GetComponent<Rigidbody2D>();
+       // if (animator.)
+	
+	}
+	
+	// Update is called once per frame
+	void Update () {
+        int horizontal = 0;
+        
+
+        horizontal = (int)(Input.GetAxisRaw("Horizontal"));
+
+        transform.position += new Vector3(horizontal, 0, 0) * speed * Time.deltaTime;
```

Comment:
It should work better in FixedUpdate, but if you can't find a nice way to use Force then that's ok